tests/dummy/app/controllers/components-examples/flexberry-menu/settings-example.js

```diff
@@ -161,6 +168,12 @@ export default Ember.Controller.extend({
       settingDefaultValue: 'undefined',
       bindedControllerPropertieName: 'currentItem.itemsAlignment'
     });
+    componentSettingsMetadata.pushObject({
+      settingName: 'collapseMenuOnItemClick',
+      settingType: 'boolean',
+      settingDefaultValue: false,
```

Comment:
As i understand default value of property is `true` in accordance with property declaration, is't it?